### PR TITLE
Move `<Steps />`, `<Tabs />`, `<Tab />`, `<Cards />`, `<Card />`, `<FileTree />` components to `nextra/components`

### PR DESCRIPTION
**Pull request by tristandubbeld**
To enable usage of the Tabs, Tab and Steps components in docs, blog and custom themes, we extract it to the nextra package components folder and export it from there.

## In this PR
- [x] Moved Tabs and Tab component from nextra-theme-docs to nextra/components.
- [x] Moved Steps component from nextra-theme-docs to nextra/components.
- [x] Updated the blog and docs examples.
- [x] Updated the Nextra documentation.
- [x] Installed `@headlessui/react` in the nextra package.
- [x] Move scrollbar styles.

fixes https://github.com/shuding/nextra/issues/1676

### CODE REVIEW

File: .changeset/wise-chicken-relax.md
Comment (tristandubbeld) [inline .changeset/wise-chicken-relax.md:1]: Something got screwed with syncing my fork with origin, which I think caused this changeset to appear. Can I just remove it and add a new one @B2o5T?

Comment (B2o5T) [reply .changeset/wise-chicken-relax.md:1]: yeah, I removed you can add a new one

File: packages/nextra/src/components/tabs.tsx
Comment (tristandubbeld) [inline packages/nextra/src/components/tabs.tsx:1]: This file includes a `className` `scrollbar`. This class only seems to exist in the `docs`. Should we move it somewhere else? What do you think is a good way to structure this @B2o5T?

Comment (B2o5T) [reply packages/nextra/src/components/tabs.tsx:1]: do you talk about `nextra-scrollbar`? you can move it `nextra/styles`

**Comment by tristandubbeld** [comment]
Hey @B2o5T, anything else needed to get this merged? I'll be gone for the next few weeks and might have a few minutes to squeeze in some lost changes now. Let me know :)

**Comment by B2o5T** [comment]
@tristandubbeld sorry for the delay, I'll rebase/merge it soon

**Comment by B2o5T** [comment]
@tristandubbeld it was more challenging than I expected, I moved also `Cards`, `Card` and `FileTree` components. Please try locally if I nothing missed 😅

localhost_3000_posts_callout (1)

localhost_3000_posts_callout

**Review by B2o5T — APPROVED** [review]

**Comment by tristandubbeld** [comment]
Hey, awesome job @B2o5T! I didn't have the time to check it out earlier but it all looks good. Glad this got merged!

**Comment by B2o5T** [comment]
@tristandubbeld ty! Enjoy

**Pull request merged**